### PR TITLE
[#161886901] - Implement 429 return code.

### PR DESCRIPTION
**Pull request by chiadav**
[#161886901]- Implement 429 return code.

### CODE REVIEW

**Comment by digitalcitizenship** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  Please include a Pivotal story at the beginning of the PR title (see below).
  </td>
    </tr>
  
<tr>
      <td>:warning:</td>
      <td>

  Please include a description of your PR changes.
  </td>
    </tr>
  </tbody>
</table>



  Example of PR titles that include pivotal stories:

  * single story: `[#123456] my PR title`
  * multiple stories: `[#123456,#123457,#123458] my PR title`

    
<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Comment by cloudify** [comment]
@chiadav I didn't notice before merging, but the story ID is wrong, the story is https://www.pivotaltracker.com/story/show/157896066

File: api/public_api_v1.yaml [hunk 1]
```diff
@@ -144,6 +144,8 @@ paths:
           description: Unauthorized
         '404':
           description: No service found for the provided ID.
+        '429':
+          description: Too many mequests
```
Comment (gunzip) [inline api/public_api_v1.yaml]: typo (this states for every othre dscription in thi pr)

Comment (chiadav) [reply api/public_api_v1.yaml]: ok

Comment (cloudify) [reply api/public_api_v1.yaml]: thanks @gunzip, @chiadav please create a new PR for fixing the typos, thanks

**Pull request merged**